scaffolds/widget/widgetModelBinder.ts

```diff
@@ -0,0 +1,34 @@
+import { widgetName } from "./constants";
+import { IModelBinder } from "@paperbits/common/editing";
+import { WidgetModel } from "./widgetModel";
+import { Contract } from "@paperbits/common";
+import { WidgetContract } from "./widgetContract";
+
+/**
+ * This is a class that helps to prepare the model using data described
+ * in the contract.
+ */
+export class WidgetModelBinder implements IModelBinder<WidgetModel> {
+    public canHandleContract(contract: Contract): boolean {
+        return contract.type === widgetName;
+    }
+
+    public canHandleModel(model: any): boolean {
+        return model instanceof WidgetModel;
+    }
+
+    public async contractToModel(contract: WidgetContract): Promise<WidgetModel> {
+        const model = new WidgetModel();
+        // model.property = contract.property;
```

Comment:
maybe remove this comment or this is on purpose?

Reply:
yes, this is scaffold, so the comment is on purpose

Reply:
I think it's on purpose. It's good to have it and we follow the same in other files